spec/features/hub/send_messages_spec.rb

```diff
@@ -11,8 +11,10 @@
           :intake,
           preferred_name: "Tobias",
           email_address: "tfunke@example.com",
+          email_notification_opt_in: "yes",
           phone_number: "+14155551212",
-          sms_phone_number: "+14155551212"
+          sms_phone_number: "+14155551212",
+          sms_notification_opt_in: "yes"
```

Comment:
huh, were we allowing people to email clients who hadn't opted in? maybe i'm not understanding these tests

Reply:
Turns out yes, and we can discuss that with program/product/design at some point.